### PR TITLE
Fix traverse doc

### PR DESCRIPTION
**Pull request by 1ven**

### CODE REVIEW

File: src/traverse.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ var sequence = require('./sequence');
  * then uses [`sequence`](#sequence) to transform the resulting Traversable of Applicative
  * into an Applicative of Traversable.
  *
- * Dispatches to the `sequence` method of the third argument, if present.
+ * Dispatches to the `traverse` method of the third argument, if present.
```
Comment (davidchambers) [inline src/traverse.js]: Should this be `fantasy-land/traverse` rather than `traverse`?

Comment (1ven) [inline src/traverse.js]: @davidchambers I think no, as [map](https://github.com/ramda/ramda/blob/v0.24.1/src/map.js#L45) is dispatching both, `map` and `fantasy-land/map` but docs are mentioning just `map` method.
As I correctly understand, we need to dispatch `traverse` method along with `fantasy-land/traverse` like it's done in `map`

Comment (paldepind) [inline src/traverse.js]: The docs state that the function supports Fantasy Land traversables. I don't think it's necessary to mention that it dispatches to `fantasy-land/traverse`. Can't we consider that to be more of an implementation detail? Users should just have to think about the abstractions.

**Comment by paldepind** [comment]
LGTM 👍

**Pull request merged**